### PR TITLE
keepassxc: Persist configuration file

### LINKED ISSUE #2018: keepassxc doesn't persist properly
I am not sure what would cause this behaviour, but if you upgrade from KeepassXC 2.4.0 to 2.4.1 with scoop, your persisted ini `keepassxc.ini` will be renamed to `keepassxc.ini.original` (but it will remain as `keepassxc.ini` in the `persist/keepassxc` folder), and a new `keepassxc.ini` will be installed.

The manifest is clearly asking to persist the ini, so I'm not sure what's going on here (is it KeepassXC itself managing the settings?)

### PR DESCRIPTION
**Pull request by pigsflew**
Persist doesn't seem to work properly (see lukesampson/scoop#3212) for now, so doing it manually

- Closes #2018

### CODE REVIEW

File: bucket/keepassxc.json [hunk 1]
```diff
@@ -17,9 +17,20 @@
...
+        "keepassxc-cli.exe",
+        "keepassxc-proxy.exe"
     ],
-    "persist": "keepassxc.ini",
+    "installer": {
+        "script": [
+            "if (!(Test-Path \"$persist_dir\\keepass.ini\")) {",
+            "    New-Item \"$persist_dir\\keepass.ini\" -Force | Out-Null",
+            "}",
+            "Copy-Item \"$persist_dir\\keepass.ini\" \"$dir\\keepass.ini\" -Force",
```
Comment (Ash258) [inline bucket/keepassxc.json]: Do not copy empty file just do it in else branch

File: bucket/keepassxc.json [hunk 2]
```diff
@@ -17,9 +17,20 @@
...
+        ]
+    },
+    "uninstaller": {
+        "script": "Copy-Item \"$dir\\*\" \"$persist_dir\" -Include 'keepass.ini' -Force"
```
Comment (Ash258) [inline bucket/keepassxc.json]: Include is not needed for one file

```suggestion
        "script": "Copy-Item \"$dir\\keepass.ini\" \"$persist_dir\" -Force"
```

File: bucket/keepassxc.json (hunk 1 above)
Comment (Ash258) [reply bucket/keepassxc.json]: No need to specify file in copy target

File: bucket/keepassxc.json (hunk 2 above)
Comment (linsui) [reply bucket/keepassxc.json]: And please check if it exist.

File: bucket/keepassxc.json [hunk 3]
```diff
@@ -17,9 +17,23 @@
     },
     "bin": [
         "KeePassXC.exe",
-        "keepassxc-cli.exe"
+        "keepassxc-cli.exe",
+        "keepassxc-proxy.exe"
     ],
-    "persist": "keepassxc.ini",
+    "installer": {
+        "script": [
```
Comment (Ash258) [inline bucket/keepassxc.json]: Make it one line. It is short enough for Github PR 

image

Comment (pigsflew) [reply bucket/keepassxc.json]: Done

File: bucket/keepassxc.json [hunk 4]
```diff
@@ -17,9 +17,15 @@
     },
     "bin": [
         "KeePassXC.exe",
-        "keepassxc-cli.exe"
+        "keepassxc-cli.exe",
+        "keepassxc-proxy.exe"
     ],
-    "persist": "keepassxc.ini",
+    "installer": {
+        "script": "if (Test-Path \"$persist_dir\\keepass.ini\") {  Copy-Item \"$persist_dir\\keepass.ini\" \"$dir\" -Force }"
```
Comment (linsui) [inline bucket/keepassxc.json]:
```suggestion
        "script": "if (Test-Path \"$persist_dir\\keepass.ini\") { Copy-Item \"$persist_dir\\keepass.ini\" \"$dir\" -Force }"
```

**Pull request merged**